backend/controllers/userControllers.js

```diff
@@ -58,6 +64,15 @@ const authUser = asyncHandler(async (req, res) => {
   const { email, password } = req.body;
 
   const user = await User.findOne({ email });
+<<<<<<< HEAD
```

Comment:
what is this please fix this 
![image](https://github.com/user-attachments/assets/bebd705c-2248-456d-a36f-6318931db405)